daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-template.md

```diff
@@ -0,0 +1,90 @@
+---
+type: docs
+title: Multi-app template file
+linkTitle: Multi-app Run template
+weight: 2000
+description: Unpack the multi-app template file and its variables
+---
+
+{{% alert title="Note" color="primary" %}}
+ Multi-app `dapr run -f` is currently a preview feature only supported in Linux/MacOS. 
+{{% /alert %}}
+
+The multi-app template file is a single YAML configuration file that you can use to configure multiple applications alongside a Dapr sidecar. Execute the following command for Dapr to parse the multi-app template file, named `dapr.yaml` by default:
+
+```cmd
+dapr run -f
+```
+
+To name the multi-app template file something other than `dapr.yaml`, run:
+
+```cmd
+dapr run -f ./<your-preferred-file-name>.yaml
+```
+
+The multi-app template file can include any of the following parameters. 
+
+```yaml
+version: 1
+common: # optional section for variables shared across apps
+  resourcesPath: ./app/components # any dapr resources to be shared across apps
+  env:  # any environment variable shared across apps
+    - DEBUG: true
+apps:
+  - appID: webapp # required
+    appDirPath: .dapr/webapp/ # required
+    resourcesPath: .dapr/resources # (optional) can be default by convention
+    configFilePath: .dapr/config.yaml # (optional) can be default by convention too, ignore if file is not found.
+    appProtocol: HTTP
+    appPort: 8080
+    appHealthCheckPath: "/healthz" # All _ converted to - for all properties defined under daprd section
```

Comment:
Where did this comment come from 
"# All _ converted to - for all properties defined under daprd section"

Reply:
the example in one of the code PRs... should I remove the comment?